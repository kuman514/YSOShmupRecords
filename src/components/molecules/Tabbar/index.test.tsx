import { cleanup } from '@testing-library/react';
import { beforeEach, describe, it } from 'vitest';

describe('Tabbar', () => {
  beforeEach(() => {
    cleanup();
  });

  it('should show tabs just as provided', () => {});

  it('should have tabs that are all clickable', () => {});

  it('should show first tab active at first', () => {});

  it('should switch active tab when click a tab', () => {});
});
