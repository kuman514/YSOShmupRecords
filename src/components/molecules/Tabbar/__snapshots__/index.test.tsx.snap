// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Tabbar > should highlight tab button corresponding to current tab index 1`] = `
.c1 {
  all: unset;
  box-sizing: border-box;
  font-size: 16px;
  font-weight: 500;
  padding: 8px 16px;
  color: var(--white-color);
  background-color: transparent;
  border: none;
  border-radius: 16px;
  cursor: pointer;
}

.c1:hover {
  background-color: var(--hovering-color);
  color: var(--white-color);
}

.c1:disabled {
  border: none;
  color: var(--gray-color);
  background-color: transparent;
  text-decoration: line-through;
}

.c0 {
  width: 100%;
  min-height: 50px;
  display: flex;
  flex-direction: row;
}

<div
  class="c0"
>
  <button
    class="c1"
    style="min-height: 100%; border-radius: 0; text-align: center; word-break: keep-all; flex-grow: 1;"
  >
    Koishi Komeiji
  </button>
  <button
    class="c1"
    style="min-height: 100%; border-radius: 0; text-align: center; border-top: 3px solid transparent; word-break: keep-all; flex-grow: 1;"
  >
    Hoshino Takanashi
  </button>
  <button
    class="c1"
    style="min-height: 100%; border-radius: 0; text-align: center; word-break: keep-all; flex-grow: 1;"
  >
    Yasuo
  </button>
</div>
`;
